spec/services/data_migrations/delete_work_history_audits_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe DataMigrations::DeleteWorkHistoryAudits do
+  describe '#change' do
+    it 'enques jobs to DeleteAuditsWorker' do
+      create(
+        :application_work_history_break_audit,
+        user_type: nil,
+        user_id: nil,
+        created_at: DateTime.new(2024, 9, 3, 12),
+        action: 'create',
+        username: '(Automated process)',
+      )
+
+      expect { described_class.new.change }.to change(
+        DeleteAuditsWorker.jobs, :size
+      ).by(1)
+    end
+  end
+
+  describe '#relation' do
+    it 'returns the audits that need deleting' do
+      audit_1 = create(
```

Comment:
Descriptive names for the audit variations would make it easier to understand the differences